data/helpers.d/nginx

```diff
@@ -64,8 +29,6 @@ ynh_add_nginx_config () {
         ynh_replace_string --match_string="^#root_path_only" --replace_string="" --target_file="$finalnginxconf"
     fi
 
-    ynh_store_file_checksum --file="$finalnginxconf"
```

Comment:
Hmpf re-reading this, we can't just handle the checksum inside ynh_add_config because we keep editing the damn config afterwards because of thos #root_path_only etc ... I guess we should handle these in ynh_add_config as well ...

Reply:
Or maybe handle `root_path_only` before calling this script?